dowhy/gcm/confidence_intervals_cms.py

```diff
@@ -0,0 +1,70 @@
+"""This module provides functionality to estimate confidence intervals via bootstrapping the fitting and sampling.
+
+Functions in this module should be considered experimental, meaning there might be breaking API changes in the future.
+"""
+
+from functools import partial
+from typing import Union, Callable, Any, Dict
+
+import numpy as np
+import pandas as pd
+
+from dowhy.gcm.cms import ProbabilisticCausalModel, InvertibleStructuralCausalModel, StructuralCausalModel
+from dowhy.gcm.fitting_sampling import fit
+
+# A convenience function when computing confidence intervals specifically for causal queries. This function
+# bootstraps only sampling. Use bootstrap_training_and_sampling when you want to bootstrap training too.
```

Comment:
The difference between training and sampling is not clear here. What is being trained, and then later, what is being sampled?

Reply:
Revised the explanation and example:
https://github.com/py-why/dowhy/pull/438/commits/b267fff7e2cf67916fc9521626d8c8428b38f298#diff-6e9d52ae849c95f6c3e58abdf4b6f2fd2d06245eb3388915d5324e9e12a2d3a6R15